Dockerfile

```diff
@@ -1,5 +1,6 @@
 # Golang version defined in https://github.com/kaishuu0123/erd-go/blob/${ERD_VERSION}/go.mod#L3
-ARG ERD_GOLANG_VERSION=1.15
+ARG ERD_GOLANG_BUILDER_TAG=1.15-alpine
```

Comment:
This line change is unrelated to the pull request. But since we don't track this version with an `updatecli` manifest, no problem to change it like this for homogeneity